frontend/javascripts/viewer/controller/scene_controller.ts

```diff
@@ -474,9 +497,7 @@ class SceneController {
   }
 
   setClippingDistance(value: number): void {
-    // convert nm to voxel
-    const voxelPerNMVector = getVoxelPerUnit(Store.getState().dataset.dataSource.scale);
-    V3.scale(voxelPerNMVector, value, this.planeShift);
+    this.clippingDistance = value;
```

Comment:
Previously, this value was scaled by the voxel per unit scale and then unscaled (or something like this) in the shader again. Now this value is kept directly in the scene controller to avoid too many number conversions.

Reply:
cool :+1: could you rename `value` to `valueInUnit` or sth like this to make clear that it's not vx space?

Reply:
sounds good :+1: